From 6131abb23fc2990f9a869ef3d9c63bb0fd5ff464 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Thu, 5 Nov 2020 23:09:15 +0200
Subject: [PATCH 15/24] [libcxx] Have lexically_normal return the path with
 preferred separators

Differential Revision: https://reviews.llvm.org/D91179

(cherry picked from commit 513463fd266f059864ce3c0236494cced5de0f56)
---
 libcxx/src/filesystem/operations.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libcxx/src/filesystem/operations.cpp b/libcxx/src/filesystem/operations.cpp
index 88a039f85021..a3b93b594a07 100644
--- a/libcxx/src/filesystem/operations.cpp
+++ b/libcxx/src/filesystem/operations.cpp
@@ -1719,6 +1719,7 @@ path path::lexically_normal() const {
   if (NeedTrailingSep)
     Result /= PS("");
 
+  Result.make_preferred();
   return Result;
 }
 
-- 
2.25.1

